google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -304,6 +304,7 @@ def _api_client_factory(storage, config, parallel):
     """
     client_config = config.get_api_quota_configs()
     client_config['domain_super_admin_email'] = config.get_gsuite_admin_email()
+    client_config['excluded_resources'] = config.get_excluded_resources()
     asset_count = 0
     if config.get_cai_enabled():
         asset_count = cloudasset.load_cloudasset_data(storage.session, config)
```

Comment:
It may be worth adding a todo or other comment that when CAI supports resource ancestory / full name in metadata, then this function should filter out the excluded resources as well. Or if CAI supports exclusions in the API call, then the exclusions should be passed in there.